doc/source/structures/misc/time.rst

```diff
@@ -32,6 +32,8 @@ Special variable TIME
 
     Any time you perform arithmetic on **TIME** you get a result back that is also a :struct:`TimeSpan`. In other words, TIME is a :struct:`TimeSpan`, but TIME + 100 is also a :struct:`TimeSpan`.
 
+.. highlight:: none
+
     Note that Kerbals do not have the concept of "months"::
```

Comment:
Sphinx throws an error here. Apparently it thinks that 'Note that kerbals....' belongs to the highlight section, because it is indented. As a result this section of the file is not added to the output html.